scripts/mkcloud

```diff
@@ -196,6 +196,8 @@ function sshrun()
     cat > $mkcconf <<EOF
         export drbdnode_mac_vol=$drbdnode_mac_vol ;
         export cloud=$virtualcloud ;
+        # hostname of NFS server with repos and images:
```

Comment:
The latest change make absolute no sense. You added a comment above the line in a section where nowhere else are comments. Why you didn't make it correctly and added it to the usage? It's relatively the same amount of work.